index.html

```diff
@@ -50,46 +50,36 @@
   <!-- End Matomo Code -->
 </head>
 <body>
-	<nav>
-		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
-			<span class="newsletter-date">July-October, 2019</span>
-      <a class="language-menu--item" href="/" itemprop='url'>
-        <span itemprop="name">English</span>
-      </a>
-      <a class="language-menu--item" href="/es/" itemprop='url'>
-        <span itemprop="name">Español</span>
-      </a>
-		</div>
-    <div class="primary-nav" itemscope itemtype="http://schema.org/Organization">
-      <a class="logo-btn" href="/" itemprop="url">
+  <header>
+  	<nav itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
```

Comment:
Rethinking this structure a bit, I think it will help clear up the semantics if we move the `nav` outside the `header`.  `header` elements can contain navigation, but it should relate the navigation of the page content. In our case the `header` is the introduction to the `article` and the `nav` is global, it doesn't just apply to the page.